README.md

```diff
@@ -6,10 +6,9 @@
 [![License](https://img.shields.io/npm/l/eslint-config-twilio.svg?style=square)](./LICENSE)
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat)](#contributors-)
 
-****_NOTE: This is an initial draft.
-All thoughts, issues, and pull requests are welcome!_**
+**_All thoughts, issues, and pull requests are welcome!_**
 
-Twilio's approach to JavaScript styling. It combines both the ESLint rules and Prettier to create an opinionated set of styling and linting rules to use with your JavaScript and TypeScript projects.
+Twilio's approach to JavaScript styling. It combines both the ESLint rules and Prettier to create an opinionated set of styling and linting rules to use with your JavaScript and TypeScript projects. There is also a [Twilio Style CLI](https://github.com/twilio-labs/twilio-style-cli) to help projects onboard.
```

Comment:
Maybe move that to a new line and have a `_Note_: ... ` so that it stands out?

Reply:
updated!